subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -1060,7 +1060,7 @@ static int lpn_timeout_status(const struct bt_mesh_model *model, struct bt_mesh_
 }
 
 const struct bt_mesh_model_op bt_mesh_cfg_cli_op[] = {
-	{ OP_DEV_COMP_DATA_STATUS,   BT_MESH_LEN_MIN(5),     comp_data_status },
+	{ OP_DEV_COMP_DATA_STATUS,   BT_MESH_LEN_MIN(0),      comp_data_status },
```

Comment:
Should it be `BT_MESH_LEN_MIN(1)`? (I see that the opcode is only one octet here instead of two, so 0 might be correct due to this?)
![image](https://github.com/user-attachments/assets/b06f2c7c-14aa-4964-82f2-d616644e72be)

Reply:
Looks like you are right, I think the opcode length is taken into account already so it's my mistake here. I mistakenly went from thinking "empty page data" to "empty status" but forgot the page number.